mihaela-etropolska/Homework4/quadratic-equation.php

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title></title>
+</head>
+<body>
+	<?php
+		$a = 15;
+		$b = -2;
+		$c = 0;
+		$d = $b * $b - 4 * $a * $c;
+
+		if ($d > 0) {

```

Comment:
@mihaela-etropolska 
You need to handle the case where "$d" is equal to 0. Please add it and resubmit :+1: